packages/web-app-files/src/composables/actions/files/useFileActionsDownloadArchive.ts

```diff
@@ -68,6 +91,12 @@ export const useFileActionsDownloadArchive = ({ store }: { store?: Store<any> }
         label: () => {
           return $gettext('Download')
         },
+        disabledTooltip: ({ resources }) => {
```

Comment:
@kulmann this could potentially become more elaborate (say: "different messages for different reasons of being disabled), but not needed _yet_